apps/api/src/app/insights/insights.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller, Get, Query, UnauthorizedException } from '@nestjs/common';
+import { ApiOperation, ApiQuery } from '@nestjs/swagger';
+import { FeatureFlagsService } from '@novu/application-generic';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
+import { UsageInsights } from './usecases/usage-insights/usage-insights.usecase';
+import { UsageInsightsCommand } from './usecases/usage-insights/usage-insights.command';
+
+@Controller({
+  path: 'insights',
+})
+export class InsightsController {
```

Comment:
This is a temporary controller protected allowed to be triggered by novu admins (protected by LD) to trigger individual insight emails during testing period